components/search_results_item/search_results_item.jsx

```diff
@@ -222,8 +230,44 @@ class SearchResultsItem extends React.PureComponent {
     }
 
     render() {
-        const {post, channelIsArchived} = this.props;
-        const channelName = this.getChannelName();
+        const {post, fileInfo, channelIsArchived, channelType} = this.props;
+        let channelName = this.getChannelName();
+
+        if (fileInfo) {
+            channelName = this.props.channelName;
+            if (channelType === Constants.DM_CHANNEL) {
+                channelName = this.props.intl.formatMessage({
+                    id: 'search_item.file_badge.direct_message',
+                    defaultMessage: 'Direct Message',
+                });
+            }
+            if (channelType === Constants.GM_CHANNEL) {
+                channelName = this.props.intl.formatMessage({
+                    id: 'search_item.file_badge.group_message',
+                    defaultMessage: 'Group Message',
+                });
+            }
+            return (
+                <div
+                    data-testid='search-item-container'
+                    className='search-item__container'
+                >
+                    <FileSearchResultItem
+                        fileInfo={this.props.fileInfo}
+                        teamName={this.props.currentTeamName}
+                        channelDisplayName={channelName}
+                        onClick={() => this.setState({showPreview: this.props.fileInfo.postId})}
```

Comment:
`showPreview` sounds like it's a boolean, but we're actually storing a `postId` here. Can we pick another name?

Reply:
Actually it can be boolean, I'll change it.

Reply:
Done